### PR TITLE
Ens Profile

### PR DESCRIPTION
**Pull request by derHowie**
Figma link (if any): https://www.figma.com/file/hzfwKOddfRmeVO4KUsG9wM/Rainbow-BX?type=design&node-id=8127-263102&mode=design&t=wI4jh5AIavjFb4OG-0

## What changed (plus any additional context for devs)
ENS info will now appear when a name has an associated eth address

## Screen recordings / screenshots
PoW: https://recordit.co/iqI2DXTFGz

## What to test
Ensure the profile attributes referenced in the design above appear in NFT details. Note that this branch does not include the expiry changes.

### CODE REVIEW

File: src/entries/popup/pages/home/NFTs/NFTDetails.tsx [hunk 1]
```diff
@@ -95,6 +98,26 @@ export default function NFTDetails() {
...
+    enableProfile: nft?.familyName === 'ENS',
+  });
+  console.log({
+    ensAddress,
+    ensName,
+    ensBio,
+    ensCover,
+    ensTwitter,
+    ensWebsite,
+  });
```
Comment (greg-schrammel) [inline src/entries/popup/pages/home/NFTs/NFTDetails.tsx:120]:
```suggestion

```

**Review by greg-schrammel — APPROVED** [review]

File: src/entries/popup/pages/home/NFTs/NFTDetails.tsx [hunk 2]
```diff
@@ -326,6 +349,78 @@ const NFTPriceSection = ({ nft }: { nft?: UniqueAsset | null }) => {
...
+              label={i18n.t('nfts.details.ens_address')}
+              value={truncateAddress(address as AddressOrEth)}
+              valueSymbol={'doc.on.doc'}
+              onClick={() => copyAddress(address as Address)}
+            />
+          )}
+          {twitter && (
+            <NFTInfoRow
+              symbol={'person.crop.rectangle.fill'}
+              label={'Twitter'}
```
Comment (estebanmino) [inline src/entries/popup/pages/home/NFTs/NFTDetails.tsx:401]: shouldn't this be `X` now? same for the birdIcon

Comment (derHowie) [reply src/entries/popup/pages/home/NFTs/NFTDetails.tsx:401]: never

Comment (derHowie) [reply src/entries/popup/pages/home/NFTs/NFTDetails.tsx:401]: jk, idk. I thought the same, but we call it Twitter in settings still, so I figured I'd stick with the design for now.

**Comment by derHowie** [comment]
> copy hint is out of screen when copying a long id
> 
>  Screen.Recording.2023-11-29.at.11.55.35.mov

added a fix for this and for the trait description issue you mentioned on Slack

**Review by estebanmino — APPROVED** [review]

**Pull request merged**